log/src/test/java/kcl/teamIndexZero/traffic/log/fileIO/FileInputTest.java

```diff
@@ -24,7 +25,7 @@ public void tearDown() throws Exception {
 
     @Test
     public void testRead() throws Exception {
-        FileOutput out = new FileOutput("", "testRead.test");
+        kcl.teamIndexZero.traffic.log.fileIO.FileOutput out = new kcl.teamIndexZero.traffic.log.fileIO.FileOutput("", "testRead.test");
```

Comment:
After having a look too I am unable to draw the cause of these, as I never edited the Log and other files. Let me know what can be done to make sure it is changes, won't touch it myself as of now because I might end up making it worse, so just tell me what to do in this, I will delete that specific line only.